javascript/features/killtime/killtime_manager.test.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
```

Comment:
Do we need those tests, afaik it is all covered via the _commands.test?

Reply:
I see this as a seperate unit testing the things the manager does. (As in most of our JS code)

Here we test whether the handling of the weapons works. In commands we test if the handling of the command works. 
As I don't think we have fakes commands is technically an IntegrationTest but I still much like to test the unit's logic seperately.